scripts/deploy-chart-test.sh

```diff
@@ -54,7 +54,7 @@ CUSTOM_HELM_REPO_LOCAL_NAME="${CUSTOM_HELM_REPO_LOCAL_NAME:-"local_repo"}"
 split-container-image() {
   local INPUT="$1"
   CONTAINER_IMAGE_REPOSITORY="${INPUT%:*}"
-  CONTAINER_IMAGE_TAG="${INPUT#*:}"
+  CONTAINER_IMAGE_TAG="${INPUT##*:}"
```

Comment:
This is a tiny fix for registry URLs than contain a port. I use `kind` with a local registry for the local development (following this [guide](https://kind.sigs.k8s.io/docs/user/local-registry/)). 

Example of a container image registry address: `localhost:5001/vector:test`

Current `split-container-image` function exports 
```bash
CONTAINER_IMAGE_REPOSITORY=localhost:5001/vector
CONTAINER_IMAGE_TAG=5001/vector:test
```

I changed it to pick the value after the last colons sign entry.
```bash
CONTAINER_IMAGE_REPOSITORY=localhost:5001/vector
CONTAINER_IMAGE_TAG=test
```

Reply:
Ah, thanks for this - definitely one of the places where things have **Just Worked**™️ and I don't think about it 😅